examples/just-the-basics/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@example/just-the-basics",
```

Comment:
Thoughts on `@example/basic` vs `@example/just-the-basics`? Slightly cleaner, IMO.

Reply:
I realize having `starter`, `basic`, and `minimal` will be slightly confusing, but we're racing towards removing `starter` so that won't be the case for long.

Reply:
I debated this **for so long**. I liked using `just-the-basics` to match the `create-astro` select option we'll introduce. If we wanna change it, fine by me 👍

Reply:
I'm +1 for a more terse folder / package name, but not opposed to the display name being `Just the Basics` if you prefer it.

Reply:
No prob!